Tests/Tests/CDTReplicationTests.m

```diff
@@ -26,6 +26,37 @@
 #import "TD_Revision.h"
 #import "TDPuller.h"
 #import "TDPusher.h"
+#import "AllNullResponseURLProtocol.h"
+
+@interface ChangesFeedRequestCheckInterceptor : NSObject <CDTHTTPInterceptor>
+
+@property (nonatomic) BOOL changesFeedRequestMade;
+
+@end
+
+@implementation ChangesFeedRequestCheckInterceptor
+
+- (instancetype)init
+{
+    self = [super init];
+    if (self) {
+        _changesFeedRequestMade = NO;
+    }
+    return self;
+}
+
+- (CDTHTTPInterceptorContext *)interceptRequestInContext:(CDTHTTPInterceptorContext *)context
+{
+    NSURL *url = context.request.URL;
+
+    if ([[url path] containsString:@"/_changes"]) {
```

Comment:
Add a comment to show that this tests the fact that the interceptors are being run on the changes tracker as that is the only code path which accesses the `_changes` URL